website/src/pages/showcase.js

```diff
@@ -5,46 +5,49 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
-import Layout from '@theme/Layout';
-
+import React, {useEffect, useState} from 'react';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import ThemedImage from '@theme/ThemedImage';
+
+import {Section} from './index';
+import IconExternalLink from '../theme/Icon/ExternalLink';
 
 const renderApp = (app, i) => {
+  const imgSource = !app.icon.startsWith('http')
+    ? useBaseUrl('img/showcase/' + app.icon)
+    : app.icon;
+
   return (
-    <div className="showcase" key={i}>
-      <div>
-        <a href={app.infoLink}>{renderAppIcon(app)}</a>
-        <h3>{app.name}</h3>
-        {renderLinks(app)}
-        {renderInfo(app.infoTitle, app.infoLink)}
+    <div className="showcase" key={`app-${app.name}-${i}`}>
+      <div className="iconBox">
+        <img src={imgSource} alt={app.name} className="iconBackground" />
+        <img src={imgSource} alt={app.name} className="icon" />
+      </div>
+      <div className="showcaseContent">
+        <div>
+          <h3>{app.name}</h3>
+          {renderLinks(app)}
+        </div>
+        {app.infoLink && (
+          <a
+            className="articleButton"
+            href={app.infoLink}
+            target="_blank"
+            title={app.infoTitle}>
+            Learn more{' '}
```

Comment:
Have we thought about removing these learn more links? They often lead to really stale content

Reply:
those that were really stale have been addressed (ex. we replaced the Discord link with the new blogpost about Android).

I think that it's important to have those links that will lead people to the companies' own words, talks, etc as a way to "back" the fact that they are in the showcase. Like, I honestly prefer when there's a "learn more" button so that I can actually hear/read what the company has to say around RN and their usage (which is also something that annoys me about MSFT apps 🤣 we only have talks at confs and no blogpost to link 😰 will work on changing that in the future I swear)

Reply:
Can we remove the "Learn More" links for Facebook, Ads Manager. As well, can we remove Instagram as an app? We can probably add Messenger Desktop and use https://reactnative.dev/blog/2021/08/26/many-platform-vision#expanding-to-new-platforms as the link?

Reply:
I will remove the links in next commit, but if you can suggest the changes for the listed apps it would be nice.